sdk/communication/tests.yml

```diff
@@ -64,4 +64,10 @@ stages:
     EnvVars:
       SKIP_PHONENUMBER_INTEGRATION_TESTS: TRUE
       SKIP_LIVE_TEST: TRUE
+      AZURE_SUBSCRIPTION_ID: $(acs-subscription-id)
```

Comment:
Is this a merge problem? I dont think we need to be adding these variables back in

Reply:
Yes. These changes are no necessary. These parameters are read from Subscription configuration now. See line 18-24 in https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/tests.yml